monitoring/src/monitors/chain_monitor.rs

```diff
@@ -0,0 +1,164 @@
+use std::time::Instant;
+
+use serde::Serialize;
+
+// cycle is used to measure time
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Cycle {
+    // cycle id
+    id: usize,
+    // number of downloaded block headers per cycle
+    headers: usize,
+    // number of downloaded block operatios per cycle
+    operations: usize,
+    // number of applied blocks
+    applications: usize,
+    // skip serialization to JSON for ws
+    #[serde(skip_serializing)]
+    // timestamp for fisrt block header occurrence
+    start: Instant,
+    // time to download headers and operations for cycle
+    duration: Option<f32>,
+}
+
+// monitoring statistics for bootraping
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ChainMonitor {
+    // number of block per cycle
+    blocks_per_cycle: usize,
+    // cycle is used to measure time
+    chain: Vec<Cycle>,
+}
+
+impl ChainMonitor {
+    pub fn new() -> Self {
+        Self {
+            blocks_per_cycle: 4096,
```

Comment:
You should use [associated constants](https://doc.rust-lang.org/edition-guide/rust-2018/trait-system/associated-constants.html) here.

Reply:
Fixed in 8acf122